app/views/effective/datatables/_bulk_actions_column.html.haml

```diff
@@ -4,3 +4,7 @@
   %input{value: 0, type: 'hidden'}
   %input.custom-control-input{name: "#{column[:input_name]}[]", id: "datatable_bulk_actions_resource_#{id}", value: id, type: 'checkbox', autocomplete: 'off', 'data-role': 'bulk-action' }
   %label.custom-control-label{for: "datatable_bulk_actions_resource_#{id}" }
+
+  = effective_form_with(url: '#', method: :post, id: nil, class: 'effective-datatables-form') do |form|
```

Comment:
Not sure if this is the right partial to put this form in 🤔

Reply:
This is the only part I'm concerned about. This will render a form in every single row of the bulk actions column. This is probably bad. I try to avoid any kind of forms, or form fields in the entire gem because then the datatable can be rendered inside a <form></form> without submitting weird values.

you'll notice for the search & filters use a form tag, but remove the name value from everywhere.


Instead of the form here, could this be rewritten with an ajax action, as commented above?

Reply:
The issue with an Ajax is that while it will pass the ids fine it won't redirect to another page, which as the goal.